src/UI/controllers/HTMLTooltipUIController.js

```diff
@@ -44,8 +44,13 @@ export class HTMLTooltipUIController extends UIController {
         super()
         this._options = options
         this._htmlTooltipOptions = Object.assign({}, defaultOptions, htmlTooltipOptions)
-        window.addEventListener('pointerdown', this, true)
-        window.addEventListener('pointerup', this, true)
+        // Use pointerup to mimic click behaviour when we're in the top-frame webview
+        if (options.device.globalConfig.isTopFrame) {
+            window.addEventListener('pointerup', this, true)
+        } else {
+            // Pointerdown is needed here to avoid self-closing modals disappearing because this even happens in the page
+            window.addEventListener('pointerdown', this, true)
+        }
```

Comment:
And this is the fix itself. Basically, we keep the `pointerup` event for native apps pulldown (when it's rendered in the top frame), but use `pointerdown` when we render in the page.

I think I could have used `click` instead of `pointerup` but didn't want to mess up the existing behaviour or spend time testing. The upside would just be a more _usual_ event type, so 🤷‍♂️.